Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -321,6 +331,27 @@ - (void)hideBanner
 #endif
 }
 
+- (void)_minimumHideDelayReached
+{
+    _blockHideBanner = NO;
+    if (_hideBannerAfterMinimumHideDelayReached) {
+        [self hideBanner];
+        _hideBannerAfterMinimumHideDelayReached = NO;
+    }
+}
```

Comment:
I think this also needs to be inside a `#if ENABLE(FULLSCREEN_DISMISSAL_GESTURES)`?

Reply:
Unnecessary after the new time-delta approach.